cmake/onnxruntime_unittests.cmake

```diff
@@ -876,9 +876,9 @@ if (MSVC)
                 "$<$<NOT:$<COMPILE_LANGUAGE:CUDA>>:/wd26451>")
   target_compile_options(onnxruntime_test_all PRIVATE "$<$<COMPILE_LANGUAGE:CUDA>:SHELL:--compiler-options /wd4244>"
                 "$<$<NOT:$<COMPILE_LANGUAGE:CUDA>>:/wd4244>")
-  # Avoid the error for Win arm64 Release build. error C1128: number of sections exceeded object file format limit: compile with /bigobj
+  # Avoid the error for Win arm64 build. error C1128: number of sections exceeded object file format limit: compile with /bigobj
   string(TOLOWER ${onnxruntime_target_platform} GEN_PLATFORM)
-  if (${GEN_PLATFORM} STREQUAL "arm64" AND "${CMAKE_BUILD_TYPE}" STREQUAL "Release")
+  if (${GEN_PLATFORM} STREQUAL "arm64")
```

Comment:
wonder if we still need this if we have
https://github.com/microsoft/onnxruntime/blob/643ed14720d4e92e29ec58d277f63ba1e1a2abe9/cmake/onnxruntime_common.cmake#L76
and
https://github.com/microsoft/onnxruntime/blob/643ed14720d4e92e29ec58d277f63ba1e1a2abe9/cmake/onnxruntime_common.cmake#L70
now

Reply:
It still happens for arm64 cross build for RelWithDebugInfo. It's for test code not the Common project.